EESSI-pilot-install-software.sh

```diff
@@ -172,7 +172,7 @@ else
     echo_green ">> MODULEPATH set up: ${MODULEPATH}"
 fi
 
-for eb_version in '4.7.2'; do
+for eb_version in '4.8.0'; do
```

Comment:
Keep the `4.7.2` as well for now, since we haven't switched to `4.8.0` for all toolchains yet...

To avoid confusion, we should move the `eessi-2023.06-eb-4.7.2-2021b.yml` easystack file to a subdirectory (maybe just `easystacks/2023.06/`?), so we only have one easystack file per generation of easyconfigs...

Reply:
How do I best do that because when I did this:
```
for eb_version in '4.7.2', '4.8.0'; do
```
The bot failed and the failed job was not copied to the shared directory. This is the job id, 7014, but I cannot get to it.

Reply:
How do I best do that because when I try to add more than one easybuild version it fails but the results are not copied to the shared bot-build-logs.